### DISCUSSION TITLE
Token Gif No Longer Works in Phantom Wallet

### DISCUSSION THREAD

**Discussion by wjone005** [phantom/docs#330] [Help & Support] [answered]
### Summary

The following token https://solscan.io/token/HwPtbFpd3VTe3tfyosoVtPf9WPuSk5gAKkN5xp6Npump#metadata icon which is a gif, use to execute its animation within a mobile Phantom wallet (iOS and Android). This is no longer the case. 

image: https://ipfs.io/ipfs/QmV2KAJNcQnUwFWpoCMwdt2ATwkdCmcwmJmSVtfYuPdzvW

The animation only seems to work within the web app of a Phantom wallet only.

### Example

N/A

### Steps to Reproduce

Gif animation is not appearing on mobile phantom wallets.

### Phantom Version

Version 24.27.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@wjone005 Thank you for reporting! This is a known bug that we will address in an upcoming release.
